sm/engine/dataset.py

```diff
@@ -88,6 +97,13 @@ def save(self, db, es, queue=None):
         if queue:
             queue.publish({'ds_id': self.id, 'status': self.status}, SM_DS_STATUS)
 
+    def save_acq_geometry_from_file(self, db, ms_file_path):
```

Comment:
@iprotsyuk can we rename it to "import_acq_geometry_from_file"

Reply:
yep, sounds better